src/Network/Curl.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Friendica\Network;
+
+
+/**
+ * A content class for Curl call results
+ */
+class Curl
```

Comment:
I moved the `$cur_*` parameters out of `App`, because:
- they only get set at Network::curl
- there is a dependency between the variables and Network::* methods
- there is no dependency between `App` and the variables